src/components/Orders/Orders.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { OrderItem } from '../Orders/OrderItem';
+import './Orders.scss';
+
+const Orders = (props) => {
+
+   const { currentOrders } = props;
+
+   return (
+      <div className="orders-container">
+         <div className="container">
+            {currentOrders.length > 0 ?
```

Comment:
You can use this approach
```suggestion
            {currentOrders.length ?
```